admin/class-admin-apple-meta-boxes.php

```diff
@@ -188,6 +306,9 @@ public function register_assets( $hook ) {
 			return;
 		}
 
+		wp_enqueue_style( $this->plugin_slug . '_meta_boxes_css', plugin_dir_url(
+			__FILE__ ) .  '../assets/css/meta-boxes.css' );
```

Comment:
Plugin style is currently like this. Leaving it as-is for consistency, addressing later in #96